src/views/blocks/tickets/quantity-number.php

```diff
@@ -8,42 +8,44 @@
  *
  * See more documentation about our Blocks Editor templating system.
  *
- * @link {INSERT_ARTICLE_LINK_HERE}
+ * @link    {INSERT_ARTICLE_LINK_HERE}
  *
- * @since 4.9
- * @since 4.10.8 Tweaked logic for unlimited maximum quantity allowed.
+ * @since   4.9
+ * @since   4.10.8 Tweaked logic for unlimited maximum quantity allowed.
+ * @since   TBD The input's "max" is now always set.
  *
- * @version 4.11.0
+ * @version TBD
  *
- * @var Tribe__Tickets__Ticket_Object $ticket
+ * @var Tribe__Tickets__Ticket_Object    $ticket
  * @var Tribe__Tickets__Editor__Template $this
  */
 
 $must_login = ! is_user_logged_in() && $ticket->get_provider()->login_required();
-$ticket     = $this->get( 'ticket' );
+
+$ticket = $this->get( 'ticket' );
 
 /** @var Tribe__Tickets__Tickets_Handler $handler */
-$handler      = tribe( 'tickets.handler' );
-$max_quantity = $handler->get_ticket_max_purchase( $ticket->ID );
+$handler = tribe( 'tickets.handler' );
+
+$max_at_a_time = $handler->get_ticket_max_purchase( $ticket->ID );
 
 $classes = [ 'tribe-tickets__item__quantity__number' ];
+
 if ( $must_login ) {
 	$classes[] = 'tribe-tickets__disabled';
 }
 ?>
 <div
-<?php tribe_classes( $classes ); ?>
+	<?php tribe_classes( $classes ); ?>
 >
 	<input
 		type="number"
 		class="tribe-common-h3 tribe-common-h4--min-medium tribe-tickets-quantity"
 		step="1"
 		min="0"
-		<?php if ( -1 !== $max_quantity ) : ?>
-			max="<?php echo esc_attr( $max_quantity ); ?>"
-		<?php endif; ?>
+		max="<?php echo esc_attr( $max_at_a_time ); ?>"
 		value="0"
 		autocomplete="off"
 		<?php disabled( $must_login ); ?>
 	/>
-</div>
+</div>
```

Comment:
:no_entry_sign: File must end with a newline character

<sub>hash: c43f2cf0632a5210668e5b2016415759</sub>

Reply:
@sc0ttkclark I thought we _shouldn't_ have new lines at the end of files?

Reply:
```suggestion
</div>

```

I generally have seen this used everywhere in the code as a standard, might as well